contractcourt/anchor_resolver.go

```diff
@@ -121,22 +122,33 @@ func (c *anchorResolver) Resolve() (ContractResolver, error) {
 		}
 	}
 
-	var anchorRecovered bool
+	// Create a resolver report which will record the outcome of
+	// this resolver.
+	var report = &channeldb.ResolverReport{
+		OutPoint: c.anchor,
+		Amount: btcutil.Amount(
+			c.anchorSignDescriptor.Output.Value,
+		),
+	}
```

Comment:
I think it can be worth looking into the potentially unification of this report and `c.currentReport`. Both more or less serve the same purpose, but just in different phases.